common/culture/cultures/wc_jinyu.txt

```diff
@@ -6,6 +6,11 @@
 	language = language_mogu
 	martial_custom = martial_custom_male_only
 	traditions = {
+		tradition_loyal_soldiers
+		tradition_fishermen
+		tradition_legalistic
+		tradition_mendicant_mystics
+		tradition_zealous_people
```

Comment:
I would add/replace something with tradition_sorcerous_metallurgy since they're known for water smithing.